### PR TITLE
Amplitude Extract source documentation

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## What do these changes do?

> Added documentation for Amplitude Cloud Extract source

## Type of documentation

- [x] New documentation
- [ ] Documentation update
- [ ] Hotfix

### CODE REVIEW

File: docs/sources/extract/amplitude.mdx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+You can choose the Amplitude data you want to ingest by selecting the required resources:
+
+<img src="../../assets/cloud-extract-sources/amplitude-connection-settings-2.png" alt="Selecting the data to import" />
+
+The below table mentions the syncs and [API endpoints](https://www.docs.developers.amplitude.com/analytics/#api-references) supported by these resources from Amplitude to your warehouse destination:
+
+| Resource | Full Refresh sync | Incremental sync | Primary key | Amplitude API endpoint |
+| :---| :---- | :---- | :---- | :---- |
+| `events` |  Yes | Yes | `uuid` |  `export` |
```
Comment (ameypv-rudder) [inline docs/sources/extract/amplitude.mdx:74]: @a-rampalli please verify if all the fields in this table are correct, especially the Amplitude endpoints.

**Review by nidhilashkari17 — CHANGES_REQUESTED** [review]
@a-rampalli We need to cleanup the spec file by removing airbyte doc links so that they don't show up in the UI. Can you build an updated image with these changes.
@ameypv-rudder Once Aasish updates the image, please update the connection-settings image.

**Review by a-rampalli — APPROVED** [review]
lgtm

**Review by nidhilashkari17 — APPROVED** [review]

File: docs/sources/extract/amplitude.mdx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  color={'blueviolet'}
+/>
+
+## Getting started
+
+To set up Amplitude as a source in RudderStack, follow these steps:
+
+1. Log into your [RudderStack dashboard](https://app.rudderstack.com/).
+2. Go to **Sources** > **New source** > **Cloud Extract** and select **Amplitude** from the list of sources.
+3. Assign a name to your source and click on **Next**.
```
Comment (ishachopra0401) [inline docs/sources/extract/amplitude.mdx:27]:
```suggestion
3. Assign a name to your source and click **Continue**.
```

**Review by ishachopra0401 — APPROVED** [review]

**Pull request merged**